src/tests/unittests/test_dataset_processing.py

```diff
@@ -9,11 +9,11 @@
 class TestDataProcessing(unittest.TestCase):
 
     def test_slice_dataset(self):
-        dataset = qcodes.tests.data_mocks.DataSet1D()
+        dataset = qcodes.tests.legacy.data_mocks.DataSet1D()
```

Comment:
Since we use this method (and the `DataSet2D`) a lot and it is legacy in qcodes, maybe import `qcodes.tests.legacy.data_mocks.DataSet1D` into `qtt.data` and use `qtt.data.DataSet1D` everywhere?

(we can also ignore this for now and do it once it comes up again)

Reply:
I don't want a dependency between QTT production code (qtt.data) and QCoDeS test code. As an alternative I do an explicit import of DataSet1D and DataSet2D where they are used in our unit tests. In the tests we use DataSet1D instead of using the legacy path everywhere.